server/cmd/edgemesh-server/app/config/config.go

```diff
@@ -49,7 +49,7 @@ func NewEdgeMeshServerConfig() *EdgeMeshServerConfig {
 			ContentType: constants.DefaultKubeContentType,
 			QPS:         constants.DefaultKubeQPS,
 			Burst:       constants.DefaultKubeBurst,
-			KubeConfig:  "",
+			KubeConfig:  constants.DefaultKubeConfig,
```

Comment:
edgemesh-server do not need the kubeConfig. Actually it will use the incluster with service account  to fetch apiserver.

Reply:
Thanks for your reply. I will remove the default value

Reply:
ok